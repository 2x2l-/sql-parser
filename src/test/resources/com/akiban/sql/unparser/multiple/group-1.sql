CREATE GROUP g1; TRUNCATE GROUP g1; DROP GROUP g1;
