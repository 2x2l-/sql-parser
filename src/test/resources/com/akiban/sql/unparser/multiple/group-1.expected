[0]: CREATE GROUP G1;
[1]: TRUNCATE GROUP G1;
[2]: DROP GROUP G1;